### PR TITLE
Fixes #21444 - change subitems headers style in vertical navigation

### PR DESCRIPTION
**Pull request by amirfefer**
I've decreased font size, it shouldn't be the same as the main header size.

screenshot from 2017-11-30 15-25-06

before changing:
screenshot from 2017-11-30 15-25-33

### CODE REVIEW

File: app/views/home/_vertical_menu.html.erb
Comment (ares) [inline app/views/home/_vertical_menu.html.erb]: I see it's not the code that is related to this PR but does _(item.caption) really work? shouldn't it be N_(item.caption)?

**Comment by ares** [comment]
I like it, @rohoover :+1: or :-1: :question:

File: app/views/home/_vertical_menu.html.erb
Comment (sharvit) [inline app/views/home/_vertical_menu.html.erb]: I noticed a small indentation problem, need to add another space at line 19

**Review by sharvit — CHANGES_REQUESTED** [review]

File: app/views/home/_vertical_menu.html.erb
Comment (amirfefer) [reply app/views/home/_vertical_menu.html.erb]: The string is extracted in `Loader.rb`already, so I guess it's ok to use `_()` here.
There is another translation issue, which I solved [here](https://github.com/theforeman/foreman/pull/5049)

**Comment by waldenraines** [comment]
@amirfefer @ares @rohoover what's the status of this? Can we merge?

**Review by waldenraines — APPROVED** [review]

**Comment by Rohoover** [comment]
I'm good with it.

File: app/views/home/_vertical_menu.html.erb
Comment (ares) [inline app/views/home/_vertical_menu.html.erb]: menu_title should probably be item.caption, otherwise I'm getting wrong headers, see 
screenshot_20171212_123301

**Review by ares — CHANGES_REQUESTED** [review]
It looks alright, but there's a regression breaking headers, good to go in after fixing.

**Comment by amirfefer** [comment]
@ares Thanks,
could you have another look?

**Review by ares — APPROVED** [review]
looks good and works well

**Comment by ares** [comment]
Thanks @amirfefer, merging! Katello tests failures are unrelated, they would turn green after rebasing but since they ran all successfully before the rake task failed, I guess we can merge.

**Pull request merged**
